content/v2.0/reference/flux/stdlib/experimental/geo/filterrows.md

```diff
@@ -0,0 +1,176 @@
+---
+title: geo.filterRows() function
+description: >
+  The geo.filterRows() function filters data by a specified geographic region with
+  the option of strict filtering.
+menu:
+  v2_0_ref:
+    name: geo.filterRows
+    parent: Geo
+weight: 301
+v2.0/tags: [functions, geo]
+related:
+  - /v2.0/reference/flux/stdlib/experimental/geo/gridfilter/
+  - /v2.0/reference/flux/stdlib/experimental/geo/strictfilter/
+---
+
+The `geo.filterRows()` function filters data by a specified geographic region with
+the option of strict filtering.
+This function is a combination of [`geo.gridFilter()`](/v2.0/reference/flux/stdlib/experimental/geo/gridfilter/)
+and [`geo.strictFilter()`](/v2.0/reference/flux/stdlib/experimental/geo/strictfilter/).
+
+_**Function type:** Transformation_
+
+```js
+import "experimental/geo"
+
+geo.filterRows(
+  region: {lat: 40.69335938, lon: -73.30078125, radius: 20.0}
+  minSize: 24,
+  maxSize: -1,
+  level: -1,
+  s2cellIDLevel: -1,
+  correlationKey: ["_time"],
+  strict: true
+)
+```
+
+## Parameters
+
+The region containing the desired data points.
+_See [Region definitions](/v2.0/reference/flux/stdlib/experimental/geo/#region-definitions)._
+
+_**Data type:** Object_
+
+### minSize
+Minimum number of tiles that cover the specified region.
+Default is `24`.
+
+_**Data type:** Integer_
+
+### maxSize
+Maximum number of tiles that cover the specified region.
+Default is `-1`.
+
+_**Data type:** Integer_
+
+### level
+Desired [cell level](https://s2geometry.io/resources/s2cell_statistics.html) of grid tiles.
+Default is `-1`.
+
+_**Data type:** Integer_
+
+{{% warn %}}
+`level` is mutually exclusive with `minSize` and `maxSize` and must be less than
+or equal to `s2cellIDLevel`.
+{{% /warn %}}
+
+### s2cellIDLevel
+[S2 Cell level](https://s2geometry.io/resources/s2cell_statistics.html) in `s2_cell_id` tag.
+Default is `-1`.
+
+_**Data type:** Integer_
+
+{{% note %}}
+When set to `-1`, `geo.filterRows()` attempts to automatically detect the S2 Cell ID level.
```

Comment:
automatically detect outside of the level specified in the Level field?

Reply:
`level` and `s2cellIDLevel` are different. `level` specifies what level to use when creating grid cells. `s2cellIDLevel` specifies what cell level **was** used to generate the cell ID token. It relates to the nature of the data coming in.